standard/schema/tests/test_validate_schema.py

```diff
@@ -31,14 +31,15 @@ def object_pairs_hook(pairs):
 
 
 DRAFT_SCHEMA_PATH = path(__file__).parent / 'json-schema-draft-4.json'
```

Comment:
This constant's value needs to be updated because the file moved in another commit, but if we aren't using the constant anymore, we can just remove it.